runtime/gc_modron_standard/StandardAccessBarrier.hpp

```diff
@@ -51,6 +51,7 @@ class MM_StandardAccessBarrier : public MM_ObjectAccessBarrier
 	MM_GenerationalAccessBarrierComponent _generationalAccessBarrierComponent;	/**< Generational Component of Access Barrier */
 #endif /* J9VM_GC_GENERATIONAL */
 	MM_MarkingScheme *_markingScheme;
+	MM_Scavenger *_scavenger;
```

Comment:
This should be under
OMR_GC_MODRON_SCAVENGER
or perhaps under
J9VM_GC_GENERATIONAL that is very similar although seems to be used more in context of barriers.